modules/Calendar4You/CalendarUtils.php

```diff
@@ -441,12 +441,14 @@ function transferForAddIntoTitle($type, $row, $CD) {
 		$value = getTranslatedString($Cal_Data[1],'Calendar');
 	else
 		$value = $Cal_Data[1];
-
+	if ($CD["fieldname"] == "location")  {
```

Comment:
To accept this I would need to have the google URL configurable. Maybe you can create an individual PR with a Global Variable?